docs/apm/api.asciidoc

```diff
@@ -708,3 +709,87 @@ curl -X DELETE "http://localhost:5601/api/apm/sourcemaps/apm:foo-1.0.0-644fd5a9"
 --------------------------------------------------
 {}
 --------------------------------------------------
+
+////
+*******************************************************
+*******************************************************
+////
+
+[role="xpack"]
+[[agent-key-api]]
+=== Agent Key API
```

Comment:
I think we should probably start defaulting to "APM agent" in the docs to avoid confusion with Elastic agent. I'll open an issue to update the name of the "Agent Configuration API" as well. In addition, I think this API should be ordered alphabetically with the other APIs, but I can update that when I update the name of the Agent config API.

```suggestion
=== APM agent Key API
```

Reply:
Opened https://github.com/elastic/observability-docs/issues/1339 to follow up.

Reply:
Good point!